subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -323,6 +387,30 @@ static int job_update_rejected(struct mqtt_client *const client,
 	return -EFAULT;
 }
 
+/**
```

Comment:
It probably doesn't matter as it's a .c file, but `**` would indicate that Doxygen should include this in documentation build, and I suspect that wasn't the intention

Reply:
I used `**` just to align with the other API documented functions in this library. We should have a cleanup on that front ass well.